packages/nodejs/src/diagnose.ts

```diff
@@ -194,6 +194,52 @@ export class DiagnoseTool {
 
     return config
   }
+
+  public sendReport(data: object) {
+    const json = JSON.stringify(data)
+
+    const config = this.#config.data
+    const params = new URLSearchParams({ api_key: config["apiKey"] || "" })
+
+    const opts = {
+      port: 443,
+      method: "POST",
+      host: "staging.lol",
```

Comment:
Shouldn't this be `appsignal.com`, like it was on the previous implementation of this method?

Reply:
Yep!